### PR TITLE
support oneliner

### PR DESCRIPTION
**Pull request by littleq0903**
try `python -m docx2html.tool foo.docx`, you will get it.

### CODE REVIEW

File: docx2html/tool.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from docx2html.core import convert
+
+if __name__ == "__main__":
+    import sys
+    import_filename = sys.argv[1]
+    print sys.argv[1]
```
Comment (jlward) [inline docx2html/tool.py]: This doesn't need to be here

File: docx2html/tool.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    html = convert(import_filename)
+    new_html = open(import_filename.replace('.docx','.html'), 'w')
+    new_html.write(html)
+    new_html.close()
```
Comment (jlward) [inline docx2html/tool.py]: Here you should do:

```
with open(import_filename.replace('.docx', '.html'), 'w') as new_html:
    new_html.write(html)
```

File: docx2html/tool.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from docx2html.core import convert
+
+if __name__ == "__main__":
+    import sys
+    try:
+        import_filename = sys.argv[1]
+    except:
```
Comment (jlward) [inline docx2html/tool.py:7]: Bare excepts are dirty, try `except IndexError:`

**Comment by kylegibson-rldatix** [comment]
Do we want to print to stdout instead of creating a new file? That's the behavior I would expect.

File: docx2html/tool.py (hunk 3 above)
Comment (littleq0903) [inline docx2html/tool.py:7]: just saw this, I will make it

**Comment by littleq0903** [comment]
maybe an option for that? how about -o for stdout

**Comment by jlward** [comment]
This library is no longer being maintained. Development has been shifted to https://github.com/CenterForOpenScience/pydocx

**Comment by winhamwr** [comment]
Hi Colin,

Thanks for the pull request anyway. Are you attending Pycon again this year? We're going to make the trip to Toronto again and I'm really looking forward to it.

Thanks
-Wes

**Comment by littleq0903** [comment]
Hi Wes,

No, I won't go to PyCon this year, it's been too many events in Taiwan this year, maybe next time ;)

All the best on PyCon trip
- Colin

**Pull request closed without merging**